community: update governance (#3164)
chaos-mesh/chaos-mesh: GOVERNANCE.md

## Roles and responsibilities

### Contributor

> Contributors are community members who contribute in concrete ways to the project. Anyone can contribute to the project and become a contributor, regardless of their skillset.  There is no expectation of commitment to the project, no specific skill requirements, and no selection process. There are many ways to contribute to the project, which may be one or more of the following (but not limited to):

Reviewer: ```suggestion
Contributors are community members who contribute in concrete ways to the project. Anyone can contribute to the project and become a contributor, regardless of their skillset. There is no expectation of commitment to the project, no specific skill requirements, and no selection process. There are many ways to contribute to the project, which may be one or more of the following (but not limited to):
```

- Reporting or fixing bugs.
- Identifying requirements, strengths, and weaknesses.
- Improving the Chaos Mesh website.
- Writing documentation.
- Joining discussions on [Twitter](https://twitter.com/chaos_mesh), our community Slack channel [(#project-chaos-mesh)](https://slack.cncf.io/), or at [community meetings](https://docs.google.com/document/d/1H8IfmhIJiJ1ltg-XLjqR_P_RaMHUGrl1CzvHnKM_9Sc/edit).
- Evangelizing about the project (e.g. a link on a website or word-of-mouth awareness raising).

For first-time contributors, it’s recommended to start by going through [Contributing to Chaos Mesh](https://github.com/chaos-mesh/chaos-mesh/blob/master/CONTRIBUTING.md), and joining our community Slack channel.

> As one continue to contribute to the project and engage with community, they may at some point be eligible to become a member. 

Reviewer: ```suggestion
As one continues to contribute to the project and engage with the community,  he/she may at some point become eligible for a Chaos Mesh Member. 
```

### Member

> Chaos Mesh [Members](https://github.com/orgs/chaos-mesh/people) are continuously active contributors in the community. They may have lightweight issues and PRs assigned to them, contribute tutorials, use cases, and are expected to remain active contributors to the project.

Reviewer: ```suggestion
Chaos Mesh [Members](https://github.com/orgs/chaos-mesh/people) are continuously active contributors in the community. There are multiple ways to stay "active" and engaged with us - contributing to codes, raising issues, writing tutorials and case studies, and even answering questions. 
```

To become a Chaos Mesh member, you are expected to: 

- Have made multiple contributions, which may be one or more of the following (but not limited to):
    - Authored PRs on GitHub. 
    - Filed, or commented on issues on GitHub.
    - Join community discussions (e.g. community meetings, Slack).
- Have read [Contributing to Chaos Mesh](https://github.com/chaos-mesh/chaos-mesh/blob/master/CONTRIBUTING.md).
- Sponsored by at least 1 Chaos Mesh [maintainer or committer](https://github.com/chaos-mesh/chaos-mesh/blob/master/MAINTAINERS.md).

> Contributors that meet the above requirements will then be nominated by their sponsor at the next available [community meeting](https://docs.google.com/document/d/1H8IfmhIJiJ1ltg-XLjqR_P_RaMHUGrl1CzvHnKM_9Sc/edit) to become a member. The existing maintainers will confer and decide whether to grant membership or not. 

Reviewer: ```suggestion
Contributors that meet the above requirements will then be nominated by their sponsor at the next available [community meeting](https://docs.google.com/document/d/1H8IfmhIJiJ1ltg-XLjqR_P_RaMHUGrl1CzvHnKM_9Sc/edit) to become a member. The maintainers will confer and decide whether to grant membership or not.  Chaos Mesh members will be invited into the Chaos Mesh organization, which is recognized by the Chaos Mesh badge under the profile. Members are required to be eligible for Chaos Mesh Committers.  
```

Members are expected to respond to issues and PRs assigned to them, and be the owners of the code or docs they have contributed. Members that have not contributed to the project or community in any way in over 6 months will lose their membership.

As one gains experience and familiarity with the project and as their commitment to the community increases, they may find themselves being nominated for committership at some stage.

### Committer

Committers are active community members who have shown that they are committed to the continuous development of the project through ongoing engagement with the community. Committership allows contributors to more easily carry on with their project-related activities by giving them direct access to the project’s resources. 

> Typically, a potential committer will need to show that they have a sufficient understanding of the project, its objectives, and its strategy. To become a committer, you are expected to: 

Reviewer: ```suggestion
Typically, a potential committer needs to show that they have a sufficient understanding of the project, its objectives, and its strategy. To become a committer, you are expected to: 
```
